### DISCUSSION TITLE
some xwayland windows dont get any input nor can be focused for some reason

### DISCUSSION THREAD

**Discussion by HqNw** [hyprwm/Hyprland#1606] [General]
it started with warframe launcher and now with rofi its kind of weird because if i have brave open and open rofi on the same workspace it works fine but if i open it on an empty one it just doesnt work
edit: now warframe launcher doesnt even show

**Comment by HqNw** [comment]
downgrading to 0.21 fixes it for now but warframe is still finicky

**Reply by vaxerski** [reply]
do a `git bisect` if this is a regression

**Reply by Aqa-Ib** [reply]
xwayland-x11 changes on hyprland 3 weeks ago (3 february) broke rofi x11.
 Install the rofi wayland fork.

**Reply by vaxerski** [reply]
@Aqa-Ib you mean 668842124067f9b6513edf1c28478bec12d709a0 ?

**Reply by Aqa-Ib** [reply]
let me check the exact commit that broke it

**Reply by Aqa-Ib** [reply]
fcf1baf broke it

**Reply by Aqa-Ib** [reply]
66e3679 fixed it though, you are fast
